pkgs/top-level/all-packages.nix

```
@@ -8058,6 +8058,8 @@ with pkgs;
 
   netsniff-ng = callPackage ../tools/networking/netsniff-ng { };
 
+  nixpacks = callPackge ../applications/virtualization/nixpacks { };
```

This seems to be a typo.
```suggestion
  nixpacks = callPackage ../applications/virtualization/nixpacks { };
```